src/commands/search.ts

```diff
@@ -34,44 +36,15 @@ export class SearchCommand extends Command {
 			() => c("command-name").t`search`,
 			() => c("command-description").t`Find all information on a card!`
 		);
-		const nameSubcommand = buildLocalisedCommand(
-			new SlashCommandSubcommandBuilder(),
-			() => c("command-option").t`name`,
+		const nameSubcommand = getNameSubcommand(
 			() => c("command-option-description").t`Find all information for the card with this name.`
-		);
-		const nameOption = buildLocalisedCommand(
-			new SlashCommandStringOption().setRequired(true),
-			() => c("command-option").t`input`,
-			() => c("command-option-description").t`Card name, fuzzy matching supported.`
-		);
-		const passwordSubcommand = buildLocalisedCommand(
-			new SlashCommandSubcommandBuilder(),
-			() => c("command-option").t`password`,
+		).addStringOption(getResultLangStringOption());
```

Comment:
Noting for myself that unlike the vendorId in /price, this is optional and can/should come last, so the difference between these two is correct